spec.md

```diff
@@ -1841,6 +1845,48 @@ To disallow infinitely recursive hardware, modules cannot contain instances of
 itself, either directly, or indirectly through instances of other modules it
 instantiates.
 
+## Groups
+
+Groups are named collections of statements inside a module.  Groups contain
+optional functionality for a module that can be included or excluded without
+affecting the intended functionality of a circuit.  Groups are primarily used to
+keep verification, debugging, or FIRRTL statements, not relevant to the
+operation of the circuit, in a separate area.  Groups have special emission
+rules defined in the FIRRTL ABI.
+
+The `defgroup`{.firrtl} statement declares a group with a specific name.  A
+`defgroup`{.firrtl} may declare a group for a `circuit`{.firrtl} or for an
+`extmodule`{.firrtl}.  Group names participate specially in the module
+namespace--no module may include a suffix of `_<group-name>` for any declared
+group name.
+
+The `group`{.firrtl} statement creates a group instance within a module.  A
+`group`{.firrtl} must refer to a group declared by a `defgroup`{.firrtl} of the
+current `circtui`{.firrtl}.
```

Comment:
Circuit

Reply:
Fixed in https://github.com/chipsalliance/firrtl-spec/pull/108/commits/0a54caf039ac72960226a34c368eafeb2cc90911